src/common/currPathParams.ts

```diff
@@ -0,0 +1,9 @@
+import { matchPath } from "react-router-dom";
+
+
+export const getPathParams = (path: string, param: string) => {
```

Comment:
we're no longer using this right? We're getting from redux store instead. Probably can remove

Reply:
I though I deleted the file...let me fixit